.circleci/config.yml

```diff
@@ -4,7 +4,7 @@ aliases:
   - &test-base-legacy
     working_directory: /tmp/drafter
     environment:
-      CXXFLAGS: '-Werror -Wno-error=unused-function'
+      CXXFLAGS: '-Wno-error=unused-function'
```

Comment:
It's a little disappointing to remove Werror (although its only in `legacy` branch of CI). There are warnings when built with current/modern compiler versions.